modules/mod_articles_news/tmpl/_item.php

```diff
@@ -17,17 +17,23 @@
 	<<?php echo $item_heading; ?> class="newsflash-title">
 	<?php if ($item->link !== '' && $params->get('link_titles')) : ?>
 		<a href="<?php echo $item->link; ?>">
-			<?php echo $item->title; ?>
-		</a>
-	<?php else : ?>
-		<?php echo $item->title; ?>
-	<?php endif; ?>
-	</<?php echo $item_heading; ?>>
-<?php endif; ?>
+ 			<?php echo $item->title; ?>
+ 		</a>
+ 	<?php else : ?>
+ 		<?php echo $item->title; ?>
+ 	<?php endif; ?>
+ 	</<?php echo $item_heading; ?>>
+ <?php endif; ?>
```

Comment:
```suggestion
			<?php echo $item->title; ?>
		</a>
	<?php else : ?>
		<?php echo $item->title; ?>
	<?php endif; ?>
	</<?php echo $item_heading; ?>>
<?php endif; ?>
```

remove extra leading space